web/src/js/__tests__/components/FlowView/DetailsSpec.js

```diff
@@ -46,5 +47,8 @@ describe('Details Component', () => {
         let details = renderer.create(<Details flow={tflow}/>),
             tree = details.toJSON()
         expect(tree).toMatchSnapshot()
+        details = renderer.create(<Details flow={tflowServerAddressNull}/>),
+            tree = details.toJSON()
+        expect(tree).toMatchSnapshot()
```

Comment:
Let's move this into a seperate test, i.e. `it('should render if the server connection is missing', ...)`. That makes things clearer.